src/app/toolbar/components/toolbar/toolbar.component.ts

```diff
@@ -44,4 +52,12 @@ export class ToolbarComponent {
       verticalPosition: 'top',
     });
   }
+
+  showMessageSavingDone() {
+    this.snackBar.open(`Done`, 'OK', {
+      duration: 3000,
```

Comment:
You should stop using this magic number repeatedly. In general if you want to extend this a bit, declare an enum with duration options, (LONG, SHORT). You can also provide default duration via dependency injections:
https://material.angular.io/components/snack-bar/overview#setting-the-global-configuration-defaults